doc/releasenotes/16_0_0_summary.md

```diff
@@ -51,54 +52,21 @@
-
-#### <a id="vdiff2-ga"/>VDiff2 GA
-
-We are marking [VDiff v2](https://vitess.io/docs/16.0/reference/vreplication/vdiff2/) as Generally Available or production-ready in v16. We now recommend that you use v2 rather than v1 going forward. V1 will be deprecated and eventually removed in future releases.
-If you wish to use v1 for any reason, you will now need to specify the `--v1` flag.
-
-### <a id="tablet-throttler"/>Tablet throttler
-
-The tablet throttler can now be configured dynamically. Configuration is now found in the topo service, and applies to all tablets in all shards and cells of a given keyspace. For backwards compatibility `v16` still supports `vttablet`-based command line flags for throttler ocnfiguration.
-
-It is possible to enable/disable, to change throttling threshold as well as the throttler query.
-
-See https://github.com/vitessio/vitess/pull/11604
-
-### <a id="inc-backup"/>Incremental backup and point in time recovery
-
-In [PR #11097](https://github.com/vitessio/vitess/pull/11097) we introduced native incremental backup and point in time recovery:
-
-- It is possible to take an incremental backup, starting with last known (full or incremental) backup, and up to either a specified (GTID) position, or current ("auto") position.
-- The backup is done by copying binary logs. The binary logs are rotated as needed.
-- It is then possible to restore a backup up to a given point in time (GTID position). This involves finding a restore path consisting of a full backup and zero or more incremental backups, applied up to the given point in time.
-- A server restored to a point in time remains in `DRAINED` tablet type, and does not join the replication stream (thus, "frozen" in time).
-- It is possible to take incremental backups from different tablets. It is OK to have overlaps in incremental backup contents. The restore process chooses a valid path, and is valid as long as there are no gaps in the backed up binary log content.
-
-### <a id="repl-manager-removal"/>Replication manager removal and VTOrc becomes mandatory
-VTOrc is now a **required** component of Vitess starting from v16. If the users want VTOrc to manage replication, then they must run VTOrc.
-Replication manager is removed from vttablets since the responsibility of fixing replication lies entirely with VTOrc now.
-The flag `disable-replication-manager` is deprecated and will be removed in a later release.
-
 ### <a id="breaking-changes"/>Breaking Changes
 
 #### <a id="advertised-mysql-version"/>VTGate Advertised MySQL Version
 
-VTGate now advertises MySQL version 8.0.30. This is a breaking change for clients that rely on the VTGate advertised MySQL version and still use MySQL 5.7.
+Since [Pull Request #11989](https://github.com/vitessio/vitess/pull/11989), VTGate advertises MySQL version 8.0.30. This is a breaking change for clients that rely on the VTGate advertised MySQL version and still use MySQL 5.7.
```

Comment:
it is `8.0.31`

Reply:
IIRC, we downgraded from 8.0.31 to 8.0.30 because xtrabackup didn't have support for 8.0.31 yet.

Reply:
This is the PR - https://github.com/vitessio/vitess/pull/12273

Reply:
@GuptaManan100 Correct

Reply:
xtrabackup for the specific purpose has a flag called `–no-server-version-check` this will make xtrabackup to work on 8.0.31 as well. To my knowledge there is not regression of sorts.